pkgs/development/python-modules/fastavro/default.nix

```diff
@@ -0,0 +1,46 @@
+{ 
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  check-manifest,
+  coverage,
+  codecov,
+  cython,
+  flake8,
+  numpy,
+  pytest,
+  pytestcov,
+  twine,
+  wheel,
+}:
```

Comment:
please use 
```

{ lib
, buildPythonPackage
, fetchPypi
, check-manifest
...
```
format

Reply:
sure, will do it - and I also need to fix the maintainer attribute

Reply:
Done.